CMakeLists.txt

```diff
@@ -45,6 +45,11 @@
 #   -DNO_PYVERBS=1 (default, build pyverbs)
 #      Invoke cython to build pyverbs. Usually you will run with this option
 #      is set, but it will be disabled for travis runs.
+#   -DWITH_IBDIAGS=False (default True)
```

Comment:
The context here makes it clear we need to do a consistency review of the options to the build command.  Having mis-matched with_blah versus no_foo is ugly.  We should use with_blah and with_foo and the options should be consistent: True/False or 1/0, but not mixed (although I'm ok if we accept either True or 1, but the comments and usage info should be consistent).  We'll need to watch this on future patches too so we don't get inconsistent again after we fix the consistency issues we have now.

Reply:
WITH_IBDIAGS is just some throwaway define to sequence getting the cmake and packaging done right. It is deleted later on.

But arguably -DWITH_IBDIAGS_COMPAT should be -DENABLE_IBDIAGS_COMPAT to match most of the other users. NO_PYVERBS is the mistake here.